src/components/Home/Home.js

```diff
@@ -36,13 +38,20 @@ const Home = () => {
   return(
     <main>
       <Switch>
+        {error && <h2 className='error'>{error}</h2>}
```

Comment:
You might want to do !!error to show the error and only show your other paths if !error, but maybe that's just a weird thing I've been doing